README.md

```diff
@@ -1 +1,64 @@
-## `Capstrone project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Blog app
+
+> A mobile web application where users can manage their budget: they have a list of transactions associated with a category, so that they can see how much money they spent and on what.
+
+## Built With
+
+- HTML & CSS
+- JavaScript
+- Ruby on Rails
+- Rubocop
+- Rspec
+- OOP
+
+## Live page
+
+(Visit demo page)[https://budget-app-003.herokuapp.com/]
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- copy this link `https://github.com/ShoiraTa/budget-app.git`.
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write `git clone https://github.com/ShoiraTa/budget-app.git`.
+- go to the repository folder in your command prompt.
+- run `bundle install`.
+- run `npm install`
+- create database in your postgre called `recipe_development`
+- run `rails db:migrate`
+- To run the test run `rspec` in your command prompt.
+- To run the rails server run  `rails s` in your command prompt.
+
+## Author
+
+
+👤 **Shoira**
+
+- GitHub: [@shoirata](https://github.com/shoirata)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shoira-tashpulatova-bab4a7122/)
+
+![Shoira's GitHub stats](https://github-readme-stats.vercel.app/api?username=shoirata&count_private=true&theme=dark&show_icons=true)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
```

Comment:
- [ ] Please give a credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=),

Reply:
@ShoiraTa 

- [ ] You should add this license in the license section

Reply:
@ahmetbozaci thank you for review, but this is not a license. It is a credit to the author of the design, should I still add in licence section?

Reply:
@ShoiraTa Yeah you are right. You can keep it in Acknowledgements.

Reply:
@ahmetbozaci thank you